Cargo.toml

```diff
@@ -65,7 +70,6 @@ unnecessary_safety_doc = "warn"
 unnecessary_self_imports = "warn"
 unneeded_field_pattern = "warn"
 use_debug = "warn"
-style = "warn"
```

Comment:
Why did this get removed?

Reply:
That didn't get removed, it just was moved up to the lint groups, cuz it is one.